### DISCUSSION TITLE
Lesson 4 : No module named 'solcx'

### DISCUSSION THREAD

**Discussion by shucks18** [smartcontractkit/full-blockchain-solidity-course-py#828] [Q&A]
I've tried countless amounts of time to install solcx and the terminal keeps saying "requirement already satisfied"
![image](https://user-images.githubusercontent.com/81901871/150282864-16c61e8d-2c0f-4599-8409-2294660fc637.png)

**Comment by shucks18** [comment]
should i uninstall and re-install it back or ... whut do I do

**Reply by cromewar** [reply]
Hello @shucks18 That solcx import is a vscode problem, the contract compiles anyways or not?
Did you install it using pipx?

**Reply by shucks18** [reply]
no i just copied what it was from the website 'pip install py-solc-x' and put it into the terminal

**Reply by shucks18** [reply]
it still doesn't work even after installing it on a virtual environment :(

**Reply by shucks18** [reply]
I tried doing it on a python IDE and it worked so vscode is clearly the problem
